src/sentry/integrations/pagerduty/actions/notification.py

```diff
@@ -15,7 +17,7 @@
 class PagerDutyNotifyServiceAction(IntegrationEventAction):
     id = "sentry.integrations.pagerduty.notify_action.PagerDutyNotifyServiceAction"
     form_cls = PagerDutyNotifyServiceForm
-    label = "Send a notification to PagerDuty account {account} and service {service}"
+    label = "Send a notification to PagerDuty account {account} and service {service}with {severity} severity"
```

Comment:
should this be feature flagged? i've been playing around trying to get the two versions of the label to show up locally to no avail

Reply:
feature flagged it 👍